### ISSUE TITLE
Missing Symbols

### ISSUE THREAD

**Issue by mikelerch** [internet-development/www-server-mono#2]
First of all, great font, looks very nice.

I was wondering, how come some symbols aren't included, such as `=` for example? Is there a reason for it?
When I download [V1-ServerMono-Regular.otf](https://github.com/internet-development/www-server-mono/blob/main/fonts/V1-ServerMono-Regular.otf), the set of characters is quite limited.

Thanks in advance!

**Comment by jimmylee** [comment]
Hey @mikelerch, we are aware that some symbols aren't included. It was a bandwidth issue, but we'll see if we can get to them.

**Comment by LoganDark** [comment]
![image](https://github.com/user-attachments/assets/8b1b3724-ce34-4d49-8027-e5fda0b4e8e3)

this is a bandwidth issue?

**Comment by jimmylee** [comment]
Yes, it might be embarrassing to some, but we're a small team doing these projects for fun.

**Comment by LoganDark** [comment]
> Yes, it might be embarrassing to some, but we're a small team doing these projects for fun.

oh you meant *team bandwidth*. I thought you meant computer bandwidth, lmao. (as in I thought you implemented those symbols but then just did not include them in the exports for some reason)

**Comment by jimmylee** [comment]
You're right about this and it is misleading, I'll take them out of the site when I push `0.0.3` (unless I get to them)

**Comment by jimmylee** [comment]
Took them out in this commit, will find some time to work on these between responsibilities https://github.com/internet-development/www-server-mono/commit/9193968bcbb1cab48cee16520c14891ea4bb63c5

**Comment by Achuttarsing** [comment]
`%` symbol would be amazing too!

**Comment by jimmylee** [comment]
Thanks for pointing that out!

**Comment by mikelerch** [comment]
I think another somewhat urgent one, at least for me, would be the dollar sign `$`.
Hope next time you find some time it can be added too.

**Comment by htfy96** [comment]
Appreciate all work on this awesome font. I have been trying to use it for some ASCII arts and noticed that some frequently-used characters are missing:

```
|
~
<
>
`
$
```

I believe these are the only missing pieces before people adopting this font for everyday usage

**Comment by jimmylee** [comment]
Thanks @mikelerch and @htfy96, I'll try to find some time to get those into another Server Mono release, I appreciate your feedback.

**Comment by caidanw** [comment]
@mikelerch @htfy96 @Achuttarsing @LoganDark

I've added the missing symbols in this PR #22 and I'm creating a new release now.

**Comment by Achuttarsing** [comment]
Thanks a lot ! Amazing work
